tools/fastq_provider/fastq_provider.xml

```diff
@@ -48,9 +53,11 @@
             <option value="ssh">SSH (ENA files, for EBI only)</option>
             <option value="http">HTTP endpoint (ENA files, EBI only)</option>
         </param>
-        <param name="mode" argument="-p" type="select" label="public or private, default public. Private usable by EBI staff only" help="public or private, default 'public' " optional='true'>
+        <param name="mode" argument="-p" type="select" label="public, private, or library_wise (default public). Private usable by EBI staff only" help="public, private or library_wise, default 'public' " optional='true'>
```

Comment:
If you check the [library-wise script](https://github.com/ebi-gene-expression-group/atlas-fastq-provider/blob/develop/fetchEnaLibraryFastqs.sh), you will see that it has a public/private flag just like the file-wise script. So don't want to bundle the library/file and pulic/private settings.

Reply:
Done in 71d926a53d35e802bf0ea2bc1652e703d7da5c05